src/Util.ts

```diff
@@ -111,6 +112,14 @@ export class Util {
     return data
   }
 
+  public static async getAccessChatData(chatToken: string) {
+    const { data } = await axios.post<AccessChat>(
```

Comment:
Is it `AccessChat` or just `Chat` that you access?

Reply:
It includes `access_token`, `endpoint` and some other info to access the chat room